review.md

```diff
@@ -4,11 +4,11 @@ This is a very interesting paper that researched the Deep-Q based DRL agents for
 1) Some of the agents names need to be mentioned fully: like DRL what is its full name? list all similar names in the text. You may use notation table for all of them for the readers benefit.
 2) Page 6 - Line 16: what is the usage of serialize and deserialize.
 3) Page 6 - Line 57: (b) .... until all GVs have a job. Then in (8) ... the simulation starts, why? is there a constraint, commitment, or the simulation nature? explain.
-4) Figure 1, 2, 3: for the red numbers are they the maximum number of parts? and the arrows are they the movements? define them in the text.
+4) ~~Figure 1, 2, 3: for the red numbers are they the maximum number of parts? and the arrows are they the movements? define them in the text.~~
 5) Table 1: it is name is not mentioned in the text.
```

Comment:
> 5) Table 1: it is name is not mentioned in the text.

In our paper we have a table with a list of hyperparameters and the values we used. We wrote:
_In this research, we tested the vanilla DQN agent described in [? ] and implemented some additional promising extensions to improve the agent’s performance. This section briefly describes the implemented extensions. The exact hyperparameters used for the performance can be found in 1._

So in the end, we are indeed listing values that we don't explain. This was kinda what we agreed on to avoid repeating the original paper + it wouldn't bring any other information. How could we change that ?